reader.js

```diff
@@ -7,15 +9,156 @@ const CLOSE_TIMEOUT = 10000;
 
 let WSS;
 
-function init(wss) {
+function init(server) {
     console.log('Initializing nasonov-reader');
+    const wss = new WebSocket.Server({server});
 
     WSS = wss;
     wss.on('connection', function connected(ws, req) {});
 
+    server.on('request', (request, response) => {connectToInflux(request, response)});
```

Comment:
```suggestion
    server.on('request', connectToInflux);
```

Since you're just calling the function with the same arguments.